recipes/sundials/run_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+cc -I${PREFIX}/include -L${PREFIX}/lib -o cvRoberts_dns cvRoberts_dns.c -lm -lsundials_cvode -lopenblas -lsundials_nvecserial
```

Comment:
It would be much easier if you turned that CC command into a `CMakelists.txt` so that we can just generate an NMake/Makefile and build it that way - this makes it simpler to build on Windows!